src/main/asciidoc/index.adoc

```diff
@@ -0,0 +1,34 @@
+= Spring Cloud Data Flow Samples
+
+:doctype: book
+:toc:
+:toclevels: 4
+:source-highlighter: prettify
+:numbered:
+:icons: font
+:hide-uri-scheme:
+
+:spring-cloud-stream-docs: http://docs.spring.io/spring-cloud-stream/docs/{scst-core-version}/reference/htmlsingle/index.html
+:github-code: https://github.com/spring-cloud/spring-cloud-dataflow-samples
+
+
+ifdef::backend-html5[]
+
+Version {project-version}
+
+(C) 2012-2017 Pivotal Software, Inc.
+
+_Copies of this document may be made for your own use and for distribution to
+others, provided that you do not charge any fee for such copies and further
+provided that each copy contains this Copyright Notice, whether distributed in
+print or electronically._
+
+endif::backend-html5[]
+
+// ======================================================================================
+
+[[overview]]
+
+include::overview.adoc[]
```

Comment:
The TOC doesn't appear to be including all the sub-sections correctly.

![image](https://user-images.githubusercontent.com/1562654/32259188-eca085c4-be7b-11e7-89ee-37a653b91304.png)

Reply:
hmmm I get 
![screen shot 2017-11-01 at 10 36 40 am](https://user-images.githubusercontent.com/886859/32279994-c2720d84-bef0-11e7-80b4-c50c3b0fddd7.png)

Reply:
Even on your screenshot, it doesn't look right. Notice that `Cassandra Sample` is taking over as the title. I believe all the samples should be 1-level next to `Streaming` header. Probably, for this reason, I see it differently in my build. I open the `/generated-docs/index.html` directly.

Reply:
The file is at: `..../spring-cloud-dataflow-samples/target/generated-docs/index.html`

Reply:
Yeah I fixed that. But I intended the additional level since there are 3 gemfire samples, I was trying to get everything in streaming under the same level for consistency. Also, let's us add additional samples.  It should be Streaming/Cassandra Samples/Gemfire Samples/JDBC Samples/Custom Streaming Apps.

Reply:
I'll collapse it down, maybe leave the extra level for gemfire.